shell/platform/windows/egl/context.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_WINDOWS_EGL_CONTEXT_H_
+#define FLUTTER_SHELL_PLATFORM_WINDOWS_EGL_CONTEXT_H_
+
+#include <EGL/egl.h>
+
+#include "flutter/fml/macros.h"
+
+namespace flutter {
+namespace egl {
+
+class Context {
```

Comment:
Add doc comments. It's pretty obvious from the methods, but some added ... (wait for it) ... _context_ on how this is used and what abstraction it achieves would be a plus.

Reply:
Added context to the context! 🫡